dashboard/src/components/OperatorView/OperatorCapabilityLevel.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import { CheckCircle, XCircle } from "react-feather";
+
+import "./CapabilityLevel.css";
+
+interface IOperatorCapabilitiesProps {
+  level: string;
+}
+
+const levels = {
+  "Basic Install": 1,
+  "Seamless Upgrades": 2,
+  "Full Lifecycle": 3,
+  "Deep Insights": 4,
+  "Auto Pilot": 5,
+};
```

Comment:
Perhaps something like:
```
const BASIC_INSTALL = "Basic Install";
...
const levels = {
  BASIC_INSTALL: 1,
  ...
};
```
so the keys are defined once here? (and can be referenced in tests as per my other comment).
```